sql/synth_recipes.sql

```diff
@@ -2675,7 +2675,7 @@ INSERT INTO `synth_recipes` VALUES (39924,1,0,0,0,0,88,0,0,0,0,4100,4242,14253,0
 -- INSERT INTO `synth_recipes` VALUES (39925,1,0,0,0,0,255,0,0,0,0,4100,4242,14422,0,0,0,0,0,0,0,819,834,834,834,10,3,3,3,'Noct Doublet (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (39926,1,0,0,0,0,255,0,0,0,0,4100,4242,12601,0,0,0,0,0,0,0,818,819,819,819,7,6,6,6,'Linen Robe (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (39927,1,0,0,0,0,255,0,0,0,0,4100,4242,13950,0,0,0,0,0,0,0,823,855,844,844,1,2,1,1,'Elite Beret (desynth)');
--- INSERT INTO `synth_recipes` VALUES (39928,1,0,0,0,0,255,0,0,0,0,4100,4242,12729,0,0,0,0,0,0,0,818,819,807,807,3,4,2,2,'Linen Cuffs (desynth)');
+INSERT INTO `synth_recipes` VALUES (39928,1,0,0,0,0,23,0,0,0,0,4100,0,12729,0,0,0,0,0,0,0,818,819,807,807,2,4,1,2,;'Linen Cuffs (desynth)'); -- Wikia, bg wiki no info
```

Comment:
Line 2678 - ,2,4,1,2,;'Linen Cuf Theres a badly placed character there. Also, why remove the HQ crystal?